.storybook/kinds/components.js

```diff
@@ -9,11 +9,11 @@ import contents from '../../data/contents.json';
 
 import '../style/components.scss';
 
-// We skip rendering these components for now due to rendering issues
-// let filteredComponents = components.filter(item => ['cui-column', 'cui-container', 'cui-content', 'cui-row'].indexOf(item.name) === -1);
+// Here we filer out components that might cause rendering issues or are unnessesary at this time around
+let filteredComponents = components.filter(item => ['c-column', 'c-container', 'c-content', 'c-row', 'c-field', 'c-list', 'user-repos'].indexOf(item.name) === -1);
```

Comment:
Should we just show it and add a label like 'stable' and 'beta'? Just don't forget to remove this later. Might be strange for developers if they see these components folders in src but we can't find it in storybook.

Reply:
Yeah, agreed.
We should maybe have a label for this later on.